tasks/gen/model.cr

```diff
@@ -59,6 +60,11 @@ class Gen::Model < LuckyCli::Task
     columns_are_valid?
   end
 
+  private def resource_name_not_taken
+    @error = "A model by the name '#{resource_name.camelcase}' already exists."
```

Comment:
```suggestion
    @error = "'#{resource_name.camelcase}' model already exists at #{"./src/models/#{template.underscored_name}.cr"}."
```

I think it'd be good to show *why* it thinks that it exists since it is possible the file name is just wrong. This might give them an indication as to where they need to look